flow/environment.py

```diff
@@ -344,7 +344,7 @@ def add_args(cls, parser):
         super(DefaultSlurmEnvironment, cls).add_args(parser)
         parser.add_argument(
             '--memory',
-            help=("Specify how much memory to reserve per node in GB. "
+            help=("Specify how much memory to reserve per node, e.g. \"4g\" for 4 gigabytes. "
```

Comment:
How does a user, for example, request 256MB of memory per node now?

Reply:
With `--memory=256m`.

Reply:
Maybe it would be good to include something about MB in the help message.

Reply:
I have updated the memory message.